app/domain/repositories/impl/SimpleUserRepository.scala

```diff
@@ -0,0 +1,24 @@
+package domain.repositories.impl
+
+import domain.entity.User
+import domain.repositories.UserRepository
+
+class SimpleUserRepository extends UserRepository{
+
+  override def login(username: String, password: String): User ={
+    //TODO: change this to get actual user https://trello.com/c/YH0aogAI/86-create-db-entities
+    val user = new User()
+    user.setUsername(username)
```

Comment:
@Bukarachka setters should not be used, let's remove all setters in models and add valid constructors.